keras/tests/integration_test.py

```diff
@@ -317,5 +317,52 @@ def test_serialization_v2_model(self):
     loaded_model = keras.models.load_model(output_path)
     self.assertEqual(model.summary(), loaded_model.summary())
 
+@test_combinations.run_with_all_model_types
+@test_combinations.run_all_keras_modes
+class TokenClassificationIntegrationTest(test_combinations.TestCase):
+  """Tests a very simple token classification model.
+
+  The main purpose of this test is to verify that everything works as expected when
+  input sequences have variable length, and batches are padded only to the maximum length
+  of each batch. This is very common in NLP, and results in the sequence dimension varying
+  with each batch step for both the features and the labels.
+  """
+  def test_token_classification(self):
+    np.random.seed(1337)
```

Comment:
You can improve reproducibility (useful since you check accuracy) with `keras.utils.set_random_seed`

Reply:
Done!